spec/controllers/vm_infra_controller_spec.rb

```diff
@@ -365,6 +365,7 @@
   end
 
   it 'can migrate selected items' do
+    allow_any_instance_of(Vm).to receive(:supports_migrate?).and_return(true)
```

Comment:
@skateman Is there perhaps a better way of doing the above?

Reply:
Yup, you should mock this method on the given VM:
```
allow(vm_vmware).to receive ...
```

Reply:
@skateman corrected, thanks for help! :+1: